src/vellum/workflows/nodes/displayable/conditional_node/node.py

```diff
@@ -15,7 +15,7 @@ class ConditionalNode(BaseNode):
     """
 
     class Ports(NodePorts):
-        def __call__(self, outputs: BaseOutputs, state: BaseState) -> Iterable[Port]:
+        def __call__(self, outputs: BaseOutputs, state: BaseState) -> Set[Port]:    
```

Comment:
Decided to switch from our normal pattern here to start returning `Set`s instead of `Iterable`, mostly bc we needed to define this field as a `Set` on the pydantic event instead of `Iterable`